fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraHtmlIT.java

```diff
@@ -75,7 +74,6 @@ public void testGetDatastreamNode() throws IOException {
     public void testGetContainerTemplate() throws IOException {
         final String pid = getRandomUniqueId();
         createObject(pid);
-        addMixin(pid, REPOSITORY_NAMESPACE + "Container");
```

Comment:
The method [AbstractResourceIT.addMix()](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java#L471) can now be removed.